dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -299,24 +299,42 @@ export const Metrics = () => {
   );
 };
 
-export const GrafanaNotRunningAlert = ({ className }: ClassNameProps) => {
+const useGrafanaNotRunningAlertStyles = makeStyles((theme) =>
+  createStyles({
+    heading: {
+      fontWeight: 500,
+    },
+  }),
+);
+
+export type GrafanaNotRunningAlertProps = {
+  severity?: AlertProps["severity"];
+} & ClassNameProps;
+
+export const GrafanaNotRunningAlert = ({
+  className,
+  severity = "warning",
+}: GrafanaNotRunningAlertProps) => {
+  const classes = useGrafanaNotRunningAlertStyles();
+
   const { grafanaHost, prometheusHealth } = useContext(GlobalContext);
   return grafanaHost === undefined || !prometheusHealth ? (
-    <Alert className={className} severity="warning">
-      Grafana or prometheus server not detected. Please make sure both services
-      are running and refresh this page. See:{" "}
+    <Alert className={className} severity={severity}>
+      <span className={classes.heading}>
+        Set up Prometheus and Grafana for better Ray Dashboard experience
+      </span>
+      <br />
+      <br />
```

Comment:
```suggestion
```
Let's remove one of them? the spacing looks a bit too much.

Reply:
nvm. Just tested it. Removing it gets rid of all the space. Let's keep it as is for now for simplicity.